lib/list.gd

```diff
@@ -2376,36 +2376,22 @@ DeclareGlobalFunction( "ListWithIdenticalEntries" );
 ##
 #F  PlainListCopy( <list> ) . . . . . . . .  make a plain list copy of a list
 ##
+##  <#GAPDoc Label="PlainListCopy">
 ##  <ManSection>
 ##  <Func Name="PlainListCopy" Arg='list'/>
 ##
 ##  <Description>
+##  This operation returns a list equal to its argument, in a plain list
```

Comment:
I think it would be good to document that the returned list is guaranteed to be an actual copy (i.e. it's equal but not `IsIdenticalObj`)